### PR TITLE
KAPA-455: Katso-IdP and VTJ-exceptions

### PR DESCRIPTION
**Pull request by oj5**

### CODE REVIEW

File: conf/shibboleth/views/intercept/attribute-release.vm [hunk 1]
```diff
@@ -138,6 +138,34 @@
...
+                                            #foreach ($value in $kid.values)
+                                                <strong>$encoder.encodeForHTML($value.getDisplayValue())</strong>
+                                                <br>
+                                            #end
+                                        </td>
+                                        <input id="$kid.id" type="hidden" name="_shib_idp_consentIds"
+                                               value="$encoder.encodeForHTML($kid.id)"/>
+                                    </tr>
+                                #end
+                                #if ($vtjVerified && $vtjVerified.values.get(0).getValue() == "false")
```
Comment (jpoikela) [inline conf/shibboleth/views/intercept/attribute-release.vm]: entä jos $vtjVerified on null?

**Comment by jpoikela** [comment]
muuten :+1:

File: conf/shibboleth/views/intercept/attribute-release.vm (hunk 1 above)
Comment (oj5) [inline conf/shibboleth/views/intercept/attribute-release.vm]: http://velocity.apache.org/engine/devel/user-guide.html#Conditionals

Comment (jpoikela) [inline conf/shibboleth/views/intercept/attribute-release.vm]: ok

**Pull request merged**